core/src/components/datetime/datetime.scss

```diff
@@ -488,6 +488,32 @@ ion-picker-column-internal {
 
 // Year Picker
 // -----------------------------------
-:host(.show-month-and-year) .calendar-action-buttons ion-item {
+:host(.show-month-and-year) .calendar-action-buttons .calendar-month-year-toggle {
   --color: #{current-color(base)};
 }
+
+.calendar-month-year-toggle {
+  @include text-inherit();
+
+  position: relative;
+
+  border: 0;
+
+  background: var(--background);
```

Comment:
I've tried to account for some of the CSS variable styling APIs that existed on `ion-item` that developers _could_ be targeting via shadow parts to customize the appearance.

I think in v8 we should remove these completely in favor of the built-in properties.

e.g.:
```diff
ion-datetime::part(month-year-button) {
-  --background: blue;
+  background: blue;
}
```

If others agree, I can create the tasks and link them appropriately in the code.

Reply:
Since we're not covering _all_ the CSS variables from `ion-item`, wouldn't this be a breaking change as-is?

Reply:
I think our intention is we will target this to v8, so I can drop all the CSS variables that came from the migration from `ion-item`. 

I have a little work to get all of that ready for review 🙂

Reply:
We're making this a breaking change in v8, so it looks like the feedback here is resolved.